docs/source/install/k8s_ha.rst

```diff
@@ -46,15 +46,55 @@ makes installing the complex StackStorm infrastructure as easy as:
   # Add Helm StackStorm repository
   helm repo add stackstorm https://helm.stackstorm.com/
 
-  helm install stackstorm/stackstorm-ha
+  # Update the StackStorm repository
+  helm repo update stackstorm
 
-Once the deployment is finished, it will show you the first steps to get started working with the new cluster via WebUI
-or ``st2`` CLI client:
+  # Install StackStorm HA with an automatically-generated release name in the "stackstorm" namespace
+  # Replace "--generate-name" with a release name if you would like to name the deployment
+  # Omit the "--namespace stackstorm" flag if you would like to deploy to the "default" namespace
+  # Add "--create-namespace" if the namespace that you are specifying needs to be created
+  helm install --generate-name --namespace stackstorm stackstorm/stackstorm-ha
```

Comment:
The installation should provide the simplest way to get started:

```suggestion
  helm install --generate-name stackstorm/stackstorm-ha
```

Let's leave Helm specific flags like namespace and comments outside of the chart scope.

Reply:
Yeah, you can see the code diff suggestion and commit it that way.

I understand that's the same feedback @rush-skills gave previously.

Reply:
Oh, sorry about that, I read right over that ...